extension.json

```diff
@@ -29,7 +29,8 @@
 	},
 
 	"Hooks": {
-		"OutputPageParserOutput": "ProfessionalWiki\\PageApprovals\\EntryPoints\\PageApprovalsHooks::onOutputPageParserOutput"
+		"OutputPageParserOutput": "ProfessionalWiki\\PageApprovals\\EntryPoints\\PageApprovalsHooks::onOutputPageParserOutput",
```

Comment:
![image](https://github.com/ProfessionalWiki/PageApprovals/assets/15813104/ec55fd7c-0426-4ac7-b5eb-1cb547c907fe)
Probably comes from an older PR. But it throws an error when we access SpecialPages.

Reply:
I cannot read that error. It's to small, and if I click the image, GitHub does not show it due to auth error.

Reply:
Sometimes it does require trying a couple of times to open an image in another tab on Github. Anyway:

````
[74d5dc7a5dca37a9fcf87bbc] /index.php/Special:SpecialPages MediaWiki\Page\PageAssertionException: The given PageIdentity Special:SpecialPages does not represent a proper page

Backtrace:

from /var/www/html/includes/page/WikiPageFactory.php(57)
#0 /var/www/html/includes/context/RequestContext.php(265): MediaWiki\Page\WikiPageFactory->newFromTitle(Title)
#1 /var/www/html/includes/context/ContextSource.php(118): RequestContext->getWikiPage()
#2 /var/www/html/extensions/PageApprovals/src/EntryPoints/PageApprovalsHooks.php(15): ContextSource->getWikiPage()
#3 /var/www/html/includes/HookContainer/HookContainer.php(338): ProfessionalWiki\PageApprovals\EntryPoints\PageApprovalsHooks::onOutputPageParserOutput(OutputPage, ParserOutput)
#4 /var/www/html/includes/HookContainer/HookContainer.php(137): MediaWiki\HookContainer\HookContainer->callLegacyHook(string, array, array, array)
#5 /var/www/html/includes/HookContainer/HookRunner.php(2716): MediaWiki\HookContainer\HookContainer->run(string, array, array)
#6 /var/www/html/includes/OutputPage.php(2184): MediaWiki\HookContainer\HookRunner->onOutputPageParserOutput(OutputPage, ParserOutput)
#7 /var/www/html/includes/OutputPage.php(2232): OutputPage->addParserOutputMetadata(ParserOutput)
#8 /var/www/html/includes/OutputPage.php(2041): OutputPage->addParserOutput(ParserOutput, array)
#9 /var/www/html/includes/OutputPage.php(1967): OutputPage->addWikiTextTitleInternal(string, Title, boolean, boolean)
#10 /var/www/html/includes/OutputPage.php(4378): OutputPage->addWikiTextAsInterface(string)
#11 /var/www/html/includes/specials/SpecialSpecialpages.php(106): OutputPage->wrapWikiMsg(string, string)
#12 /var/www/html/includes/specials/SpecialSpecialpages.php(49): SpecialSpecialpages->outputPageList(array)
#13 /var/www/html/includes/specialpage/SpecialPage.php(701): SpecialSpecialpages->execute(NULL)
#14 /var/www/html/includes/specialpage/SpecialPageFactory.php(1428): SpecialPage->run(NULL)
#15 /var/www/html/includes/MediaWiki.php(316): MediaWiki\SpecialPage\SpecialPageFactory->executePath(string, RequestContext)
#16 /var/www/html/includes/MediaWiki.php(904): MediaWiki->performRequest()
#17 /var/www/html/includes/MediaWiki.php(562): MediaWiki->main()
#18 /var/www/html/index.php(50): MediaWiki->run()
#19 /var/www/html/index.php(46): wfIndexMain()
#20 {main}
```

Reply:
That looks unrelated to this PR.

Reply:
Confirmed: also present on master

Reply:
Fixed by https://github.com/ProfessionalWiki/PageApprovals/pull/18